### ISSUE TITLE
BUG: OCR with taskbone - recognized handwritten letters do not get added to frontmatter

### ISSUE THREAD

**Issue by KKassem** [zsviczian/obsidian-excalidraw-plugin#1123]
**Your environment**
SYSTEM INFO:
	Obsidian version: v1.2.7
	Installer version: v0.14.15
	Operating system: Windows 10 Pro 10.0.19044
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 1
	Plugins enabled: 1
		1: Excalidraw v1.8.26

**Describe the bug**
Running taskbone OCR on a drawing with handwritten letters, successfully copied to clipboard, but not to frontmatter.

**To Reproduce**
1. Open a new excalidraw drawing
2. Use the default drawing pen to draw a number of Latin letters
3. Run taskbone OCR on the drawing
4. Check the frontmatter of the markdown version of the drawing

**Expected behavior**
The frontmatter contains the recognized handwritten characters/letters from the drawing

**Screenshots**
Screenshot, exported .png, and raw markdown file attached

**Additional context**
Tried this on a blank vault with only excalidraw installed, to check if other plugins caused the issue.

![Drawing 2023-05-09 12 21 49 excalidraw](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/12991734/e6d3ef2e-8027-4bc3-9dcf-cbe66ab0ec7e)
[Drawing 2023-05-09 12.21.49.excalidraw.md](https://github.com/zsviczian/obsidian-excalidraw-plugin/files/11430213/Drawing.2023-05-09.12.21.49.excalidraw.md)
![taskbone_run](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/12991734/24156d4b-f251-43c6-bb27-c0afb8ebf953)
![missing_frontmatter](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/12991734/077c970e-ac3d-4c60-a8e4-9553d59c546b)

**Comment by zsviczian** [comment]
I'm just double checking: Did you run this action from the command palette and/or from the Tools Panel? I am asking because there is also an old OCR script in the script library that also works but does not copy the result to the front matter.

I tested it just now and it worked as expected for me. 

![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/14358394/dc56e4c7-b01a-4d7e-858e-64f6a900ca8a)

![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/14358394/7dce7966-a363-4d74-a67e-27e3fd8a24c2)

**Comment by KKassem** [comment]
Yes, I tried both methods (command palette and tools panel). The output is the same every time.
Unless I am doing something completely wrong, then I think it does not work for me.
I am also attaching a recreation of the steps as a .gif.
![steps](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/12991734/c6cb89d1-21c7-4649-9d2c-f0e5f92867fc)
gif.

**Comment by chadarmstrong-cbsi** [comment]
@KKassem @zsviczian I also have this issue, and stumbled upon this issue report via google. Diving in, I found that if I added a 'true' as the second parameter here: https://github.com/zsviczian/obsidian-excalidraw-plugin/blob/de5b8b64a6c208a057da5c893a88a1f1caa9304f/src/ocr/Taskbone.ts#L98
.. it started working for me, and the frontmatter text is now present. Seems like a 'force' parameter to the view.save method? Not sure of any greater issue here of course, but I could find and add to the minified plugin js. :)

**Comment by grhawk** [comment]
I have the same problem with my ipad. The ocr works but only puts the string in the clipboard and not in the frontmatter of the excalidraw note. 

Everything is up to date. Let me know if there is something i can do to assist.

**Comment by zsviczian** [comment]
I was able to reproduce... I hope I have solved it in 2.12.3

**Comment by grhawk** [comment]
Amazing! Thank you very much!!
